config/samples/image_cache/README.md

```diff
@@ -72,3 +72,107 @@ that periodically executes the `glance-cache-cleaner` utility.
 
 You can find more about image-cache configuration options in the
 [upstream](https://docs.openstack.org/glance/latest/admin/cache.html) documentation.
+
+## How to test
+
+Assuming a given `GlanceAPI` instance has been scaled up, it is possible to
+perform cache related operations and verify that `Pods` are able to see each
+other (or resolve them by `hostname`) and the image can be cached on either
+of the replicas or deleted from either of the replicas.
+
+### Step 1: Create an image
+
+The first step is to create an image to be cached at later step.
+
+```bash
+  glance --verbose image-create \
+   --disk-format qcow2 \
+   --container-format bare \
+   --name $IMAGE_NAME
+   --file myimage
+```
+
+### Step 2: Cache an image on replica 0
+
+```bash
+UUID=$(openstack image show $IMAGE_NAME -c id -f value)
+glance --os-image-url "http://${REPLICA}""0.$DOMAIN:9292" cache-queue "$UUID"
+```
+
+- **REPLICA**: a glanceAPI replicas that doesn’t own the data
+
+- **DOMAIN**: Service hostname associated with a particular GlanceAPI
+
+## Step 3: Verify that image is cached on replica 0 and not on replica 1
+
+```bash
+glance --os-image-url "http://${REPLICA}""0.$DOMAIN:9292" cache-list | awk -v img=$IMAGE_NAME '$0 ~ img {print $2}')
+# This should return 1 image in output
+glance --os-image-url "http://${REPLICA}""1.$DOMAIN:9292" cache-list | awk -v img=$IMAGE_NAME '$0 ~ img {print $2}')
+# This should return 0 image in output
+```
+- **IMAGE_NAME**: the name of the image box that will be backed by the imported
+  data
+
+- **REPLICA**: a glanceAPI replicas that doesn’t own the data
+
+- **DOMAIN**: Service hostname associated with a particular GlanceAPI
```

Comment:
Can we add here that these commands assume there are at least 2 replicas available for the same API?

Reply:
Ack